engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -493,6 +514,20 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
```

Comment:
Can we add one more level in hirarchy? e.g. ErrorEvent. As logfile information is populated only in case of ContainerErrorEvent and OperatorErrorEvent. For other events the fields will always be blank so better skip them from fields.